src/nft/components/bag/tokenSelector/BagTokenSelectorModal.tsx

```diff
@@ -90,7 +100,16 @@ export const BagTokenSelectorModal = ({ overlayClick }: { overlayClick: () => vo
         </TitleRow>
         <TokenSelectorContainer>
           {currencies.map((currency) => {
-            return <CurrencyRow key={currency.isToken ? currency.wrapped.address : currency.name} currency={currency} />
+            return (
+              <CurrencyRow
+                key={currency.isToken ? currency.wrapped.address : currency.name}
+                currency={currency}
+                selected={
+                  (!selectedCurrency && currency.isNative) || (!!selectedCurrency && selectedCurrency.equals(currency))
```

Comment:
is the check for `!!selectedCurrency` needed?

Reply:
surprisingly yes